src/dbus_api/pool/pool_3_8/api.rs

```diff
@@ -102,3 +102,50 @@ pub fn clevis_infos_property(f: &Factory<MTSync<TData>, TData>) -> Property<MTSy
         .emits_changed(EmitsChangedSignal::True)
         .on_get(get_pool_clevis_infos)
 }
+
+pub fn encrypt_pool_method(f: &Factory<MTSync<TData>, TData>) -> Method<MTSync<TData>, TData> {
+    f.method("EncryptPool", (), encrypt_pool)
+        // Optional key descriptions of key in the kernel keyring
+        // a: array of zero or more elements
+        // b: true if a token slot is specified
+        // i: token slot
+        // s: key description
+        //
+        // Rust representation: Vec<((bool, u32), String)>
+        .in_arg(("key_desc", "a((bu)s)"))
```

Comment:
Probably should go with plural here, ```key_descs``` and same below as ```clevis_infos```.

Reply:
Resolved.